services/logs-forwarder/.lagoon.multi.yml

```diff
@@ -259,19 +261,17 @@ objects:
           ssl_version TLSv1_2
           request_timeout 60s
           slow_flush_log_threshold 90s
-          <buffer tag,time>
+          <buffer>
```

Comment:
mhh interesting, my understanding of https://docs.fluentd.org/output/elasticsearch#index_name-optional is that you need to setup the buffer with time if you use time in the indexname.
but maybe because we use the `record_modifier` on top already, this is not necessary?

Reply:
Yes that is my understanding. We don't use `index_name`, we just use `target_index_key`.

The filter that adds `index_name` is also unconditional so we don't need the `index_name` fallback.